tools/install_tools.sh

```diff
@@ -0,0 +1,282 @@
+      libsystemd-dev \
+      locales \
+      ocl-icd-libopencl1 \
+      ocl-icd-opencl-dev \
+      opencl-headers \
+      python-dev \
+      python3-pip \
+      qml-module-qtquick2 \
+      qt5-default \
+      qtmultimedia5-dev \
+      qtwebengine5-dev \
+      qtlocation5-dev \
+      qtpositioning5-dev \
+      libqt5sql5-sqlite \
+      libqt5svg5-dev \
+      screen \
+      sudo \
+      vim \
+      wget \
+      gcc-arm-none-eabi \
+      libqt5x11extras5-dev \
+      libreadline-dev
+  fi
+  # install git lfs
+  if ! command -v "git-lfs" > /dev/null 2>&1; then
+    curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | sudo bash
+    sudo apt-get install git-lfs
+    git lfs pull
+  fi
+
+  # install pyenv
+  if ! command -v "pyenv" > /dev/null 2>&1; then
+    curl -L https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer | bash
+  fi
+}
+
+install_deps_brew() {
+  if [[ $(command -v brew) == "" ]]; then
+    echo "Installing Hombrew"
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+  fi
+
+brew bundle --file=- <<-EOS
+brew "cmake"
+brew "zlib"
+brew "bzip2"
+brew "rust"
+brew "rustup-init"
+brew "capnp"
+brew "coreutils"
+brew "eigen"
+brew "ffmpeg"
+brew "glfw"
+brew "libarchive"
+brew "libusb"
+brew "libtool"
+brew "llvm"
+brew "openssl"
+brew "pyenv"
+brew "qt@5"
+brew "zeromq"
+cask "gcc-arm-embedded"
+EOS
+
+  # Build requirements for macOS
+  # https://github.com/pyenv/pyenv/issues/1740
+  # https://github.com/pyca/cryptography/blob/main/docs/installation.rst
+  rustup-init -y
+
+  export LDFLAGS="$LDFLAGS -L/usr/local/opt/zlib/lib"
+  export LDFLAGS="$LDFLAGS -L/usr/local/opt/bzip2/lib"
+  export LDFLAGS="$LDFLAGS -L/usr/local/opt/openssl@1.1/lib"
+  export CPPFLAGS="$CPPFLAGS -I/usr/local/opt/zlib/include"
+  export CPPFLAGS="$CPPFLAGS -I/usr/local/opt/bzip2/include"
+  export CPPFLAGS="$CPPFLAGS -I/usr/local/opt/openssl@1.1/include"
+  export PATH="$PATH:/usr/local/opt/openssl@1.1/bin"
+  export PATH="$PATH:/usr/local/bin"
+
+}
+
+if [[ $PLATFORM == 'Linux' ]]; then
+  install_deps_apt
+elif [[ $PLATFORM == 'Darwin' ]]; then
+  install_deps_brew
+fi
+
+if [ -z "$OPENPILOT_ENV" ] && [ -n "$RC_FILE" ] && [ -z "$CI" ]; then
+  if [[ $PLATFORM == 'Darwin' ]]; then
+    echo "export PATH=\"\$PATH:$HOME/.cargo/bin\"" >> $RC_FILE
+    export PATH="$PATH:\"\$HOME/.cargo/bin\""
+  fi
+  echo "source $OP_HOME/tools/openpilot_env.sh" >> $RC_FILE
+  source $RC_FILE
+  echo "Added openpilot_env to RC file: $RC_FILE"
+else
+  echo "[WARN] Openpilot not added to $RC_FILE"
+fi
+
+pyenv install -s 3.8.5
+pyenv global 3.8.5
+pyenv rehash
+eval "$(pyenv init -)"
+pip install --upgrade pip==20.2.4
+pip install pipenv==2020.8.13
+pipenv install --system --deploy
+git -C $OP_HOME submodule update --init
+pre-commit install
+
+if [[ `git branch --show-current` == "release2" ]]; then
```

Comment:
think this check is also excessive. the tools/ directory is stripped out of release builds anyway (see release/files_common)